src/main/java/Task.java

```diff
@@ -0,0 +1,30 @@
+abstract public class Task {
+    private final String name;
+    private boolean done;
```

Comment:
I would suggest that done be changed to isDone to make it sound more like a boolean, according to the coding standard for this course.
```suggestion
    private boolean isDone;
```